core/templates/pages/footer_js_libs.html

```diff
@@ -1,4 +1,6 @@
-<!-- NOTE: Please contact @vojtechjelinek or @srijanreddy98 before modifying the file -->
+<!-- NOTE: Please contact @vojtechjelinek or @srijanreddy98 before modifying the file.
+     This is to ensure that all footer/ header libs are properly synced across these five files:
```

Comment:
Nit: here and elsewhere, drop the space before "header".

Reply:
Done!